autoread.js

```diff
@@ -101,4 +104,23 @@ function addSpeakBox(){
 	window.addEventListener('touchstart', playHotKey, true);
 
 	sbPlay.focus();
+}
+
+function handleCheckbox(){
+	let inputElems = currentLine.getElementsByTagName("input")
+	if(inputElems.length > 0 && inputElems[0].type == "checkbox"){
+		inputElems[0].click();
```

Comment:
fails on lines with more than 1 checkbox (check the line that starts with "read mannimarco")

Reply:
resolved in commit 168d8fc